### PR TITLE
Chinese translation: quick-start/create-workspace-and-project.md

### PR DESCRIPTION
**Pull request by ckyx**
Chinese translation: quick-start/create-workspace-and-project.m

Signed-off-by: ckyx <34824376+ckyx@users.noreply.github.com>

### CODE REVIEW

File: content/zh/docs/quick-start/deploy-bookinfo-to-k8s.md [hunk 1]
```diff
@@ -1,231 +1,231 @@
...
+weight: 3040
+---
+
+[Istio](https://istio.io/)，作为一种开源服务网格解决方案，为微服务提供了强大的流量管理功能。以下是[Istio](https://istio.io/latest/zh/docs/concepts/traffic-management/)官方网站上关于流量管理的简介：
+
+*Istio 的流量路由规则可以让您很容易的控制服务之间的流量和 API 调用。Istio 简化了服务级别属性的配置，比如熔断器、超时和重试，并且能轻松的设置重要的任务，如 A/B 测试、金丝雀发布、基于流量百分比切分的概率发布等。它还提供了开箱即用的故障恢复特性，有助于增强应用的健壮性，从而更好地应对被依赖的服务或网络发生故障的情况。*
+
+KubeSphere 基于 Istio 提供了三种灰度策略，包括蓝绿部署，金丝雀发布和流量镜像。
+
+Among others, a canary release represents an effective software development strategy in which a new version is deployed for testing with the base version preserved in the production environment. This strategy will bring part of the traffic to the new version being tested and the production release takes up the rest.
```
Comment (rayzhou2017) [inline content/zh/docs/quick-start/deploy-bookinfo-to-k8s.md:16]: This file is not translated yet.

File: static/images/home/section6-left.svg [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
-            <feColorMatrix in="shadowBlurOuter1" values="0 0 0 0 0.384313725 0 0 0 0 0.807843137 0 0 0 0 0.552941176 0 0 0 0.3 0"/>
-        </filter>
-        <path id="prefix__b" d="M718 5285.175l36.198 20.9c3.404 1.965 5.5 5.596 5.5 9.526v41.798c0 3.93-2.096 7.561-5.5 9.526l-36.198 20.9c-3.403 1.965-7.597 1.965-11 0l-36.198-20.9c-3.404-1.965-5.5-5.596-5.5-9.526v-41.798c0-3.93 2.096-7.561 5.5-9.526l36.198-20.9c3.403-1.965 7.597-1.965 11 0z"/>
-    </defs>
-    <g fill="none" fill-rule="evenodd" opacity=".348" transform="rotate(21 14624.781 1011.203)">
-        <use fill="#000" filter="url(#prefix__a)" xlink:href="#prefix__b"/>
-        <use fill="url(#prefix__c)" xlink:href="#prefix__b"/>
-    </g>
-</svg>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="162" height="158" viewBox="0 0 162 158">
```
Comment (rayzhou2017) [inline static/images/home/section6-left.svg:1]: Is this file mistakenly changed?

**Comment by rayzhou2017** [comment]
@ckyx please clean up your pr and sign it off. You should only translated two md files. But there are too many files are changed.

**Comment by FeynmanZhou** [comment]
I will notify the author.

**Pull request closed without merging**